api/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -117,10 +117,16 @@
    * @since 1.25.0
    */
   @ExperimentalApi("https://github.com/grpc/grpc-java/issues/6279")
-  public T blockingExecutor(Executor executor) {
+  public T offloadExecutor(Executor executor) {
     throw new UnsupportedOperationException();
   }
 
+  @Deprecated
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/6279")
+  public T blockingExecutor(Executor executor) {
```

Comment:
Was `blockingExecutor` introduced in 1.25? Is `offloadExecutor()` going to be added into 1.25.0? If so we can just remove. @zhangkun83

Reply:
It has not made into a release, but there are internal users, so we can't simply delete it.

Reply:
There's not been a release with it, but we do have some internal users (3 apps/refs) using it. So I just left the old method to allow migrating them separately. I mention that in the PR description.